src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days < 3) {
+    result = days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
+    result = (days * 40) - 20;
+  }
+
+  if (days >= 7) {
+    result = (days * 40) - 50;
```

Comment:
move 20, 40 and 50 in the variables with names that explains what this numbers actually means